src/content/reference/web-interface/app-catalog.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Web Interface Reference: The Giant Swarm App Catalog"
+description = "What the Giant Swarm App Catalog looks like on our Web Interface and how to use it."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# Web Interface Reference: The Giant Swarm App Catalog
```

Comment:
```suggestion
# The App Catalog in the Web User Interface
```

I'd prefer a title that provides the context in a more natural way, like suggested. A structured approach with colon etc. is a bit clumsy IMO.